resources/styleheets/main.css

```diff
@@ -0,0 +1,263 @@
+body {
+  padding: 0;
+  margin: 0;
+  height: 100%;
+  width: 100%;
+  background-color: #1B1717;
+  font-family: "Almarai", sans-serif;
+}
+
+.container {
+  height: 100%;
+  display: flex;
+  color: #ccc;
+}
+
+@media (max-width: 800px) {
+  .container {
+    margin-left: 0;
+    flex-direction: column-reverse;
+  }
+}
+header {
+  height: 60px;
+  background-color: #262525;
+  color: white;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.5);
+  z-index: 1;
+  position: fixed;
+  width: 100%;
+  transition: top 0.3s;
+  display: flex;
+  align-items: center;
+}
+
+.menu {
+  font-size: 30px !important;
+  margin-left: 20px;
+  cursor: pointer;
+}
+
+main {
+  margin-top: 60px;
+  flex: 57%;
+  margin-left: 13%;
+  text-align: center;
+}
+@media (max-width: 800px) {
+  main {
+    flex: 1;
+    margin-left: 0;
+  }
+  main .centered-box {
+    max-width: 90%;
+  }
+}
+
+.chats {
+  margin-top: 60px;
+  flex: 20%;
+  text-align: center;
+  color: #ccc;
+}
+@media (max-width: 800px) {
+  .chats {
+    flex: 1;
+    margin-left: 0;
+  }
+  .chats .centered-box {
+    max-width: 90%;
+  }
+}
+
+.box {
+  padding: 20px;
+  border-radius: 25px;
+  border: 2px solid rgba(255, 255, 255, 0.5);
+  max-width: 80%;
+  color: white;
+  text-align: center;
+  margin: auto;
+}
+
+.div {
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  border-top: 1px solid #ccc;
+}
```

Comment:
why make a class for a .div it is hurting my eyes just to see.
please nest inside a class if you need to style it, and if the weight is not enough then use selector, if it is not enough yet you are allow to use a class